lib/tapioca/generator.rb

```diff
@@ -11,6 +11,18 @@ class Generator < ::Thor::Shell::Color
     sig { returns(Config) }
     attr_reader :config
 
+    sig { params(camel_cased_word: String).returns(String) }
+    def self.underscore(camel_cased_word)
```

Comment:
Can we instead move this to an actual `String` extension inside the `core_ext` folder? We'd done the same for `descendants`, so that we don't have to change the use sites.

Reply:
Ah, perfect! So, we won't be polluting the core classes. I'll make the change!